src/components/CardArt.vue

```diff
@@ -40,13 +40,24 @@ const showErrorOverlay = computed(() => {
 <template>
   <div class="art-frame">
     <div v-if="showErrorOverlay" class="overlay error">
-      <div class="shape-x"></div>
+      <div>
+        <div class="shape-x"></div>
+        <p>There was an error loading the card art.</p>
```

Comment:
Thoughts on the wording here?

Reply:
this can also occur if we get a server error from the random endpoint, I think.

> There was an error loading the next card. You should skip this one.

Reply:
<img width="528" alt="Screenshot 2025-05-10 at 2 54 13 PM" src="https://github.com/user-attachments/assets/97dff752-b0f7-45ab-8801-32311c803e6c" />
<img width="757" alt="Screenshot 2025-05-10 at 2 53 57 PM" src="https://github.com/user-attachments/assets/3bf841a3-70d8-40fa-a503-bc9f68559553" />
<img width="531" alt="Screenshot 2025-05-10 at 2 53 30 PM" src="https://github.com/user-attachments/assets/a0d9ed2e-94b5-473f-aafe-3e51bb3b1178" />